internal/js/modules/k6/browser/browser/element_handle_mapping.go

```diff
@@ -1,15 +1,33 @@
 package browser
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"strings"
+	"time"
 
 	"github.com/grafana/sobek"
 
 	"go.k6.io/k6/internal/js/modules/k6/browser/common"
 	"go.k6.io/k6/internal/js/modules/k6/browser/k6ext"
+	k6common "go.k6.io/k6/js/common"
 )
 
+const (
+	optionButton     = "button"
+	optionClickCount = "clickCount"
+	optionDelay      = "delay"
+	optionModifiers  = "modifiers"
+)
+
+const noWaitAfterOption = "noWaitAfter"
+
+var imageFormatToID = map[string]common.ImageFormat{ //nolint:gochecknoglobals
+	"jpeg": common.ImageFormatJPEG,
+	"png":  common.ImageFormatPNG,
+}
```

Comment:
Please remove `nolint` and move this to `parseElementHandleScreenshotOptions`, as it's only used there.